### PR TITLE
Feature/screen 1

### PR DESCRIPTION
**Pull request by Lyokolux**
## Description

Screen 1

## Checklist

- [x] I compare my branch with `develop`
- [x] All tests passed after running `yarn s-check && yarn lint`

### CODE REVIEW

File: src/localizations/fr.json [hunk 1]
```diff
@@ -1,5 +1,14 @@
 {
   "title": "Bienvenue",
+  "home": {
+    "title": "Bonjour,",
+    "paragraph-1": "merci de participer à cette étude qui s'intéresse à la saisie de suites alphanumériques sur Internet.",
```
Comment (johannchopin) [inline src/localizations/fr.json:5]: Please use snake_case in JSON files

File: src/pages/Home.svelte [hunk 2]
```diff
@@ -1,28 +1,16 @@
 <script lang="ts">
   import { _ } from 'svelte-i18n'
-  import { swiper } from '../stores'
+  import NextButton from '../components/NextButton.svelte'
 
-  import QuestionsForm from '../components/QuestionsForm/QuestionsForm.svelte'
+  const PARAGRAPH_AMOUNT = 6
+  const I18N_KEYS = 'home.paragraph-'
```
Comment (johannchopin) [inline src/pages/Home.svelte:6]: Don't forget to update to snake_case

File: src/pages/Home.svelte [hunk 3]
```diff
@@ -1,28 +1,16 @@
...
-    },
-    {
-      labels: ['pas cool', 'cool'],
-      grade: 4
-    }
-  ]}
-/>
\ No newline at end of file
+{#each PARAGRAPHS_KEYS as paragraphKey}
+<p>{@html $_(paragraphKey)}</p>
```
Comment (johannchopin) [inline src/pages/Home.svelte:12]: Add indentation 2 spaces

**Review by johannchopin — APPROVED** [review]
Nice ✅ Just a few changes to make but then you can directly merge 🚀

File: src/localizations/fr.json (hunk 1 above)
Comment (Lyokolux) [reply src/localizations/fr.json:5]: ✅

File: src/pages/Home.svelte (hunk 2 above)
Comment (Lyokolux) [reply src/pages/Home.svelte:6]: ✅

**Pull request merged**